storm-core/src/clj/backtype/storm/util.clj

```diff
@@ -392,6 +392,15 @@
       (.addArgument command a))
     (.execute (DefaultExecutor.) command)))
 
+(defn exists-process?
+   [process-id]
+   (let [line (if on-windows? (str "cmd /c \"tasklist /FI \"PID eq "  process-id  "\" | findstr "  process-id  "\"" )
+                              (str "ps -p "  process-id))]
+        (try-cause
+           (exec-command! line)
```

Comment:
For me the supervisor is echoing the output of the ps command a lot.  It is a pain and makes debugging things difficult in some cases. Could we do something to not have them printed except on an error?

It would also be nice if we could have something that is linux specific that makes it so we don't need to run ps, but instead use /proc/ directly?